frontend/control-center/src/pages/Connectors/index.tsx

```diff
@@ -23,7 +23,7 @@ const mapStateToProps = (state: StateModel) => ({
 const connector = connect(mapStateToProps, mapDispatchToProps);
 
 const Connectors = (props: ConnectedProps<typeof connector>) => {
-  const channels = useSelector((state: StateModel) => Object.values(allChannelsConnected(state)));
+  const {channels} = props;
   const channelsBySource = (Source: Source) => channels.filter((channel: Channel) => channel.source === Source);
```

Comment:
the code was duplicated here: we don't need to use useSelector, as we are already fetching the state in mapStateToProps.